packages/sdk-android/src/runner.ts

```diff
@@ -396,6 +397,9 @@ export const configureProject = async (c: Context) => {
     mkdirSync(path.join(appFolder, 'app/src/main/assets'));
     fsWriteFileSync(path.join(appFolder, `app/src/main/assets/${outputFile}.bundle`), '{}');
 
+    // cleanup potentially existing folders from previous builds with different appId
+    removeDirSync(path.join(appFolder, 'app/src/main/java'));
```

Comment:
@mihaiblaga89 removeDirSync can be unstable especially if you re-run same ID. can you use async variant here to be 100% sure not cross-contaminated removals happen

Reply:
switched to `cleanFolder`